### PR TITLE
Fix position card not updating when opening lp

### LINKED ISSUE #409: Position card going blank and position page not updating on opening of both long and short positions
**Describe the bug**
Position card goes blank and the position page does not show the user's position on open of both short and long positions.

**To Reproduce**

1. Open a long or short position and wait to see the "transaction confirmed" pop up
2. Check position card and page
3. See the issue of the position you just opened not showing.

**Expected behavior**
Position card and page should show user's position after a user opens a position

**Screenshots**
https://user-images.githubusercontent.com/40301524/166202945-6ff45836-932d-44fb-bed5-454f50b43068.mp4

**Additional context**

### PR DESCRIPTION
**Pull request by adam5909099**
# Task:

## Description
Fix position card not updating when opening long and short positions

Fixes #409  (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

### CODE REVIEW

**Review by yichiehliu — CHANGES_REQUESTED** [review]
The vercel failed tho. Could you check it? @adam5909099

**Comment by adam5909099** [comment]
@yichiehliu It's related to e2e tests failure. I think all opened PRs have build failure due to e2e tests.

File: packages/frontend/src/state/positions/providers.tsx [hunk 1]
```diff
@@ -95,15 +95,19 @@ export const ComputeSwapsProvider: FC = ({ children }) => {
...
       setPositionType(PositionType.LONG)
     } else if (computedSwaps.squeethAmount.isLessThan(0)) {
       setPositionType(PositionType.SHORT)
     } else setPositionType(PositionType.NONE)
   }, [computedSwaps.squeethAmount, oSqueethBal, setPositionType])
 
-  const value = useMemo(
+  useAppEffect(() => {
+    refetch()
+  }, [computedSwaps.squeethAmount, refetch])
```
Comment (yichiehliu) [inline packages/frontend/src/state/positions/providers.tsx:108]: Do we include oSqueethBal in the dep array?

Comment (adam5909099) [reply packages/frontend/src/state/positions/providers.tsx:108]: I don't think we should include it as it'll cause infinite rerendering.

Comment (yichiehliu) [reply packages/frontend/src/state/positions/providers.tsx:108]: oh yh it will 😂

**Review by yichiehliu — APPROVED** [review]

**Review by EstherSeyi — APPROVED** [review]

**Review by KMKoushik — APPROVED** [review]

**Comment by KMKoushik** [comment]
@adam5909099 Can you fix the conflicts?

**Pull request merged**